animation_nodes/nodes/rotation/convert_rotations.py

```diff
@@ -66,30 +93,99 @@ def draw(self, layout):
 
     def drawLabel(self):
         for item in conversionTypeItems:
-            if self.conversionType == item[0]: return item[1]
+            if self.conversionType == item[0]:
+                if self.useList:
+                    if item[4] == 0: return "Quaternions to Eulers"
+                    elif item[4] == 1: return "Eulers to Quaternions"
+                    elif item[4] == 2: return "Quaternions to Matrices"
+                    elif item[4] == 3: return "Matrices to Quaternions"
+                    elif item[4] == 4: return "Eulers to Matrices"
+                    elif item[4] == 5: return "Matrices to Eulers"
+                    elif item[4] == 6: return "Quaternions to Axises Angles"
+                    else: return "Axises Angles to Quaternions"
+                else:
+                    return item[1]
 
     def getExecutionCode(self, required):
         if self.conversionType == "QUATERNION_TO_EULER":
-            return "euler = quaternion.to_euler('XYZ')"
+            if self.useList:
+                return "eulers = self.QuaternionsToEulers(quaternions)"
```

Comment:
Call the function directly. `eulers = AN.nodes.rotation.c_utils.quaternionsToEulers(quaternions)`.